portal/src/Notification.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback, useState } from "react";
+import { MessageBar } from "@fluentui/react";
+
+interface NotificationProps {
+  onDismiss: () => void;
+  visible: boolean;
+  msg: string;
+}
+
+export function useNotification(): {
+  showNotification: (msg: string, delay?: number) => void;
```

Comment:
`delay` parameter is no use? In fact, this hook & component seems rather trivial, so I would prefer to just include it in the parent screen.

Reply:
Moved to parent screen